### PR TITLE
Features/add address support to example project

### PR DESCRIPTION
**Pull request by daz-codersoft**
Request for update.

### CODE REVIEW

File: Examples/Example.Domain/Addresses/Commands/CreateAddressCommand.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            };
+
+            _db.Create(address);
+            command.CreatedEntity = address;
+            
+            return Task.CompletedTask;
+        }
+    }
+
+    public class SomeService
```
Comment (steveski) [inline Examples/Example.Domain/Addresses/Commands/CreateAddressCommand.cs:50]: SomeService doesn't have to be here. It was a temporary class I bunged in to test passing items to a command queue. This should also be delete from CreateCustomerCommand.cs

File: Examples/Example.Domain/Addresses/Commands/CreateAddressCommandWithTransientDbContext.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            };
+
+            _db.Create(address);
+            command.Address = address;
+            
+            await _db.SaveChangesAsync(cancellationToken).ConfigureAwait(false);
+        }
+    }
+
+    //public class SomeService
```
Comment (steveski) [inline Examples/Example.Domain/Addresses/Commands/CreateAddressCommandWithTransientDbContext.cs:49]: No need to copy the commented code. I'll clean up the original code later

File: Examples/Example.Domain/Addresses/Commands/DoSomething.cs [hunk 3]
```diff
@@ -0,0 +1,30 @@
+﻿namespace Example.Domain.Addresses.Commands
```
Comment (steveski) [inline Examples/Example.Domain/Addresses/Commands/DoSomething.cs:1]: This file isn't required

**Comment by daz-codersoft** [comment]
I was wondering about whether to do them or not.  I thought it may be useful if you need to whip up the same test on this version so left it there.  But obviously it doesn’t do anything.

 

Thanks.

 

 



Darryl Hunter| Director | Codersoft Pty Ltd
Australia M: +61 (0)450 606 804 | E: darryl.hunter@codersoft.com.au <mailto:darryl.hunter@codersoft.com.au>  | W: www.codersoft.com.au <http://www.codersoft.com.au/>

**Pull request closed without merging**